app/views/articles/_actions.html.erb

```diff
@@ -10,14 +10,34 @@
                    image_active_path: "heart-filled.svg",
                    aria_label: t("views.reactions.like.aria_label")
                  } %>
-      <%= render partial: "articles/reaction_button",
-                 locals: {
-                   category: :unicorn,
-                   description: t("views.reactions.unicorn.title"),
-                   image_path: "unicorn.svg",
-                   image_active_path: "unicorn-filled.svg",
-                   aria_label: t("views.reactions.unicorn.aria_label")
-                 } %>
+
+     <% if FeatureFlag.enabled?(:replace_unicorn_with_jump_to_comments) %>
+        <button
+          id="reaction-butt-comment"
+          aria-label="<%= t("views.articles.jump_to_comments.aria_label") %>"
+          aria-pressed="false"
+          class="crayons-reaction crayons-reaction--comment crayons-tooltip__activator relative"
```

Comment:
I think `crayons-reaction` CSS class is causing a problem when trying to jump down to the comments section (screen recording below)

https://user-images.githubusercontent.com/6045239/191386635-cb3b6af6-5d23-4fe4-affb-074974fad8cd.mov

I think this problem is happening [due to this selector](https://github.com/forem/forem/blob/main/app/assets/javascripts/initializers/initializeArticleReactions.js#L157) adding the `onClick` functions.

Would it make sense to "duplicate the `crayons-reaction` class" (same CSS) as a new `crayons-reaction-comment` or something along those lines so the layout remains the same but so that the selector won't catch that button? Maybe other alternatives are also viable solutions, unless I'm missing something that's causing this problem locally for me 🙂

Reply:
@fdocr I absolutely can not replicate this! Do you have the latest version of this branch!?

Reply:
Whoops, yeah I thought I had the latest of the branch checked out & even restarted the app a couple times... But I just tried pulling again and things worked out as expected!